src/main/java/core/basesyntax/strategy/Strategy.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.handlers.BalanceHandler;
+import core.basesyntax.strategy.handlers.PurchaseHandler;
+import core.basesyntax.strategy.handlers.ReturnHandler;
+import core.basesyntax.strategy.handlers.SupplyHandler;
+
+public class Strategy {
```

Comment:
You should have an interface called "StrategyStorage" and this class should be the implementation of the interface